javascript/javascript1/week4/homeWork/highestLowest.js

```diff
@@ -0,0 +1,26 @@
+function HighAndLow(stringNumber) {
+  const newNumberArray = stringNumber.split(" ").map(Number);
```

Comment:
Actually, you don't need to create a new array, and to use `map`. Instead you can just update the passed property like so:
```
```suggestion
  stringNumber = stringNumber.split(" ");
```

Reply:
Got it , :)